### PR TITLE
fix: login page title and styles

### PR DESCRIPTION
**Pull request by aatoaa**
change login page form title to login, fix styles for login and register page

### CODE REVIEW

File: src/pages/login/index.tsx
Comment (CPHris) [inline src/pages/login/index.tsx:1]: I am no Tailwind wizard, can you double check line 28?

**Review by CPHris — COMMENTED** [review]
Just the one comment :)

File: src/pages/login/index.tsx
Comment (aatoaa) [reply src/pages/login/index.tsx:1]: great catch, updated

**Review by CPHris — APPROVED** [review]
Lookin´ soo crispy!

**Pull request merged**